test/check-verify

```diff
@@ -207,8 +214,10 @@ def main():
 
     # Tell any subprocesses what we are testing
     os.environ["TEST_REVISION"] = revision
+    testinfra.DEFAULT_IMAGE = context
+    os.environ["TEST_OS"] = context
```

Comment:
This is in the wrong commit, "test: Reconnect to Github when keep-alive is disconnected"